frontend/src/scene/spinner/LoadingSpinner.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'react-emotion';
+import Modal, { Content, Title } from '../../components/modal';
+const SmallModal = styled(Modal)`
+    height: 40%;
+    display: flex;
+    text-align: center;
+    div {
+        font-size: 2.5rem;
+    }
+`;
+const CustomTitle = styled(Title)`
+    font-weight: 400;
+
+    strong {
+        color: ${(props) => props.theme.main};
+    }
+`;
+
+const Spinner = styled(Content)`
+    div,
+    div:after {
+        border-radius: 50%;
+        width: 10em;
+        height: 10em;
+    }
+    div {
+        margin: 0rem auto !important;
+        font-size: 1.5rem;
+        position: relative;
+        padding: 0rem !important;
+        border-top: 1.1em solid #abb2b9;
+        border-right: 1.1em solid #abb2b9;
+        border-bottom: 1.1em solid #abb2b9;
+        border-left: 1.1em solid #ffffff;
+        -webkit-transform: translateZ(0);
```

Comment:
react-emotion automatically add vendor suffixes for you. So please leave out all of it for the sake of style

Reply:
removed